Sofascore/Sofascore/Networking/CompletionHandlerApiClient.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+struct CompletionHandlerApiClient: ApiClientProtocol {
+    
+    func getEvent(eventID: Int, completion: @escaping (Result<ApiEvent, NetworkError>) -> Void) {
+        guard let url = makeEventUrl(withID: eventID) else  {
+            completion(.failure(NetworkError.invalidURL))
+            return
+        }
+        
+        URLSession.shared.dataTask(with: url) { data, response, error in
+              
+            guard let httpResponse = response as? HTTPURLResponse, httpResponse.statusCode == 200 else {
+                completion(.failure(NetworkError.invalidResponse))
+                return
+            }
+            
+            guard data != nil else {
+                completion(.failure(NetworkError.invalidData))
+                return
+            }
+            
+            do {
+                let event = try JSONDecoder().decode(ApiEvent.self, from: data!)
```

Comment:
```suggestion
                let event = try JSONDecoder().decode(ApiEvent.self, from: data!)
```
Nemoj force unwrap, radije ostavi onaj if let, samo dodaj else u kojem ces onda vratiti odgovarajuci failure